### DISCUSSION TITLE
How to display Tiered Networks

### DISCUSSION THREAD

**Discussion by MJLevineHMK** [CMSgov/price-transparency-guide#621] [Announcements]
Where we have plans/products that have tiered-networks, i.e., tier 1=primary network that pays at the highest level, tier 2=secondary network/in-network Providers, but pays at a lower level, are we required to report the Provider Prices for each of the networks or at the primary network only?

**Comment by Ginger228** [comment]
Per the rule we are required to report all negotiated rates for in-network provider.  If the contract has different rates for the same procedure code both rates should be displayed on the MRF.  If needed the additional_field could be used to show one rate is for a Tier 1 network.

**Comment by shaselton-usds** [comment]
@MJLevineHMK @Ginger228 Is there a way that you'd want this demonstrated in the schema that would be more formalized?

**Reply by Ginger228** [reply]
@shaselton-usds 
Apologies for the delayed response. 
If the tiering is based on benefits meaning the out of pocket expense for the member, I believe a new filed is needed in the Provider Object to indicate if the TIN/NPI is a Tier provider for the plan. 

<provider_groups>
   <item>
     <providers>
        <npi>
           <item>111111111
            <npi>
            <tin>
              <type> ein </type>
             <tier>
               **<value> yes </value>**
